### ISSUE TITLE
`xplorer2` package installation is broken despite the URL being fixed here

### ISSUE THREAD

**Issue by yassen-itlabs** [teknowledgist/Chocolatey-packages#29]
`xplorer2` package installation is broken despite the download URL being fixed in this repo.

The `xplorer2` package installation script at `automatic/xplorer2/tools/chocolateyinstall.ps1` seems to have been updated here on github after a change in the download URL, but when attempting an install on my machine with `choco install xplorer2`, I get a script with the old URL (http://zabkat.com/xplorer2_lite_setup.exe) which is broken. The whole script, if [P]rinted, looks like this:

```
C:\WINDOWS\system32>choco install xplorer2
Chocolatey v0.10.15
Installing the following packages:
xplorer2
By installing you accept licenses for the packages.
Progress: Downloading xplorer2 4.3.0.2... 100%

xplorer2 v4.3.0.2 [Approved]
xplorer2 package files install completed. Performing other installation steps.
The package xplorer2 wants to run 'chocolateyinstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): P

------ BEGIN SCRIPT ------

$ErrorActionPreference = 'Stop'

$packageName = $env:ChocolateyPackageName
$url         = 'http://zabkat.com/xplorer2_lite_setup.exe'
$Checksum    = '09c8d8b225c5e677c68280f9730e8628c3aa72eff0ef7f3099c377f107db7cb8'

# silent install requires AutoHotKey
$ahkExe = 'AutoHotKey'
$toolsDir    = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
$ahkFile = Join-Path $toolsDir 'chocolateyInstall.ahk'
$ahkProc = Start-Process -FilePath $ahkExe -ArgumentList "$ahkFile" -PassThru
$ahkId = $ahkProc.Id
Write-Debug "$ahkExe start time:`t$($ahkProc.StartTime.ToShortTimeString())"
Write-Debug "Process ID:`t$ahkId"

$packageArgs = @{
  packageName    = $packageName
  unzipLocation  = $toolsDir
  fileType       = 'EXE'
  url            = $url
  softwareName   = 'xplorer² lite*'
  checksum       = $Checksum
  checksumType   = 'sha256'
  silentArgs     = '/S'
  validExitCodes = @(0)
}
Install-ChocolateyPackage @packageArgs


------- END SCRIPT -------
Do you want to run this script?([Y]es/[N]o):
```

Not sure why the updated URL is not propagated to the live repository. What am I missing? Thanks in advance for any advice.

**Comment by teknowledgist** [comment]
I apologize.  I'm not entirely sure what happened because life has been *crazy* for the last couple months.  I suspect that I learned of the problem, and made the change (March 21 according to my local repository change log) with a plan to have it fixed in the next version release with the thought that it would be soon.

I have pushed an updated package (v4.3.0.201) with the corrected download URL.  It should be available soon.
